apps/status-api/src/controllers/BlockchainController.ts

```diff
@@ -0,0 +1,37 @@
+import { Controller, Get } from '@nestjs/common'
+import { Block } from '@defichain/whale-api-client/src/api/Blocks'
+import { WhaleApiClient } from '@defichain/whale-api-client'
+
+@Controller('blockchain')
+export class BlockchainController {
+  constructor (private readonly client: WhaleApiClient) {
+  }
+
+  @Get('status')
+  async getToken (): Promise<{ [key: string]: string }> {
+    const blocks: Block[] = await this.client.blocks.list(1)
+
+    const nowEpoch = Date.now()
+    const latestBlockTime = blocks[0].time * 1000
+    const timeDiff = nowEpoch - latestBlockTime
+
+    const statusMap = new Map<string, number>([
+      ['outage', (45 * 60 * 1000)],
+      ['degraded', (30 * 60 * 1000)],
+      ['operational', 0]
+    ])
```

Comment:
I think something like this

```typescript
type BlockchainStatus = 'outage' | 'degraded' | 'operational'
const StatusMap = new Map<BlockchainStatus, number>([
      ['outage', (45 * 60 * 1000)],
      ['degraded', (30 * 60 * 1000)],
      ['operational', 0]
    ])
```
would help constraint this even further.

Personally, would prefer using normal records for `StatusMap` instead of a map object, as we are not really putting anything into map and does not as such need its utilities. Its more explicit as well.
```typescript
const StatusMap: Record<BlockchainStatus, number> = {
    'outage': 45 * 60 * 1000,
    'degraded': 30 * 60 * 1000,
    'operational': 0
}
```

Pulling it outside of the controller class would also probably be ideal since its suppose to be a constant.